pkg/cloudprovider/openstack_test.go

```diff
@@ -742,9 +745,14 @@ func TestGetNeutronPortNodeEgressIPConfiguration(t *testing.T) {
 					IPv6: "2000::/64",
 				},
 				Capacity: capacity{
-					IP: openstackMaxCapacity - 2, // 2 allowed_address_pairs configured on the port.
+					IP: openstackMaxCapacity + 3 - 2, // excluding 2 allowed_address_pairs configured on the port.
```

Comment:
Ironically, this shows that we can now get above the default / max capacity in corner cases ;-) On the other hand, I suppose this shouldn't happen in real life ... otherwise, we'd have to have a math.Max() for all the cloudprovider plugins where the absolute highest value that we report is the port's default capacity?

Reply:
yes, anyway default capacity is always `openstackMaxCapacity - len(p.AllowedAddressPairs)`, should we just ignore counting`cloudPrivateIPsCount` for OSP case ?